packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -116,9 +116,29 @@ import {
 // };
 
 import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import { Helmet } from 'react-helmet';
 import { Layout, HeaderLayout, ContentLayout } from '@strapi/design-system/Layout';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { LinkButton } from '@strapi/design-system/LinkButton';
 import { Main } from '@strapi/design-system/Main';
+import { Typography } from '@strapi/design-system/Typography';
 import adminPermissions from '../../permissions';
+import MarketplacePicture from '../../assets/images/marketplace-coming-soon.png';
```

Comment:
Can you put the png file inside the MarketPlace folder?

`admin/admin/src/pages/MarketplacePage/assets/marketplace-coming-soon.png`